scripts/provision-environment-directories.sh

```diff
@@ -89,8 +89,8 @@ copy_templates() {
 
     if [ ${filename} != "platform_locals.tf" ] && [ ${filename} != "platform_providers.tf" ] && [ ${filename} != "platform_data.tf" ] && [[ "${filename}" !=  "member_"* ]]
     then
-      if [ ${filename} == "subnet_share.tf" ] && [ ${account_type} == "member-unrestricted" ]; then
```

Comment:
Alternatively you could test if the account_type != "member"

Reply:
thanks Dave, good to know